app/mailers/enrollment_mailer.rb

```diff
@@ -23,7 +23,8 @@ class EnrollmentMailer < ActionMailer::Base
         else
           "contact@api.gouv.fr"
         end
-
+    
+      @last_message = Message.where(enrollment_id: enrollment.id).order(:created_at).last.content
```

Comment:
Note that the last message is not necessary the one send with this refusal request, hence this could be a source of bug.

As the case is quite rare, surely we can consider this implementation good enough. We should at least mention this in a comment though